src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -80,34 +81,68 @@ public class RedisHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposab
 
                 previousBroadcastTask = Task.WhenAll(tasks);
             });
+
+            channelName = _channelNamePrefix + ".AllExcept";
+            _logger.LogInformation("Subscribing to channel: {channel}", channelName);
+            _bus.Subscribe(channelName, async (c, data) =>
+            {
+                await previousBroadcastTask;
+
+                _logger.LogTrace("Received message from redis channel {channel}", channelName);
+
+                var message = DeserializeMessage<RedisExcludeClientMessage>(data);
+                var invocationMessage = (InvocationMessage)message.HubMessage;
+                var excludedIds = message.ExcludedIds;
+
+                //// TODO: This isn't going to work when we allow JsonSerializer customization or add Protobuf
```

Comment:
How is it not going to work? If we're just writing serialized data directly to the connection, then we're broken for MsgPack too, if one node is using JSON and the other is using MsgPack. If we're serializing separately for every connection, then it should work fine with JsonSerializer customization and Protobuf because each send uses the client's configured serializers.